src/block.rs

```diff
@@ -92,11 +98,13 @@ impl BlockCount {
 
 /// Represent a device holding blocks.
 pub trait BlockDevice: core::fmt::Debug {
```

Comment:
Add doc on how we're agnostic from what a block is, and how the user can define its own blocks.